plugins/org.jkiss.dbeaver.ext.databend/plugin.xml

```diff
@@ -0,0 +1,74 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<?eclipse version="3.2"?>
+
+<plugin>
+
+    <extension point="org.jkiss.dbeaver.generic.meta">
+        <meta id="databend"
+              class="org.jkiss.dbeaver.ext.databend.model.DatabendMetaModel"
+              driverClass="com.databend.jdbc.DatabendDriver"
+              dialect="databend"/>
+    </extension>
+
+    <extension point="org.jkiss.dbeaver.dataSourceProvider">
+        <datasource
+                class="org.jkiss.dbeaver.ext.databend.DatabendDataSourceProvider"
+                description="Databend"
+                id="databend"
+                parent="generic"
+                label="Databend"
+                icon="icons/databend_icon.png"
+                dialect="databend">
+            <drivers managable="true">
+                <driver
+                    id="databend"
+                    label="Databend"
+                    class="com.databend.jdbc.DatabendDriver"
+                    sampleURL="jdbc:databend://{username}:{password}@{host}:{port}[/{database}]"
+                    defaultPort="8000"
+                    icon="icons/databend_icon.png"
+                    iconBig="icons/databend_icon_big.png"
+                    description="Databend driver (server version 1.0.0+)"
+                    supportedConfigurationTypes="MANUAL,URL"
+                    webURL="https://github.com/databendcloud/databend-jdbc"
+                    categories="sql,analytic,columns">
+                    <file type="jar" path="maven:/com.databend:databend-jdbc:RELEASE[0.0.7]" load-optional-dependencies="true" bundle="!drivers.databend" />
+                    <file type="license" path="https://www.apache.org/licenses/LICENSE-2.0.txt" bundle="!drivers.databend"/>
+                    <file type="jar" path="drivers/databend" bundle="drivers.databend"/>
```

Comment:
```suggestion
                    
```

Please remove this string. Dbeaver doesn't have this path `path="drivers/databend"`